src/components/CardContainer.js

```diff
@@ -40,15 +67,43 @@ const CardContainer =  () => {
     )
   }
 
+  const selectComponents = () => {
+    if (playMode === 'play') {
+      return (
+        <div className='card-container'>
+          {createCardComponents()}
+        </div>
+      ) 
+    } else if (playMode === 'won') {
+      return (
+        <div>
+          <h2 className='notification'>You have won.</h2>
+        </div>
+      )
+    } else if (playMode === 'lost'){
+      return (
+        <div>
+          <h2 className='notification'>You have lost.</h2>
+        </div>
+      )
+    }
+  }
+
+  useEffect(() => {
+    if (playMode === 'won' || playMode === 'lost') {
+      setTimeout(() => setPlayMode('play'), 1500)
+    }
+  }, [playMode])
+
   return (
-    <div 
-        style= {{
-          display: 'flex',
-          flexFlow: 'row wrap'
-        }}>
-        {createCardComponents()}
-        <h2>{score}</h2>
-      </div> 
+    <div className='game-container'>
+      <div className='header-container'>
+        <h1 className='header'>BlotMemory</h1>
+        <ScoreBoard score={score}/>
+      </div>
+      <InfoBar />
+       {selectComponents()}
```

Comment:
My additional suggestion, on top of the stuff in pr #1, would be that if you feel like it needs to be abstracted out, then I would make it its own component. Like this whole component would be `<GameBoard>` and everything that `selectedComponents` does would be in `CardContainer`:
```jsx
// GameBoard.js

    <div className='game-container'>
      <div className='header-container'>
        <h1 className='header'>BlotMemory</h1>
        <ScoreBoard score={score}/>
      </div>
      <InfoBar />
     <CardContainer cards={cards} playMode={playMode} handleScore={handleScore} />
  </div>
``